concent_api/core/views.py

```diff
@@ -505,6 +521,40 @@ def handle_receive_delivered_force_report_computed_task(request, delivered_messa
     return ack_report_computed_task
 
 
+def handle_receive_force_subtask_results(
+    request,
+    decoded_message:                message.concents.ForceSubtaskResults,
+    previous_message_from_database: Union[ReceiveStatus, Message],
```

Comment:
Using the same parameter for both a message and a status makes things very confusing and error prone. The name says it's a message but it might be something entirely different. Please replace this with two separate arguments with distinct types. If one is specified, the other should be `None`.

Or, instead of passing two different objects just pass the key because that's the only piece of data you use and it's the same in both.

Reply:
Right, I've pick second solution. Passing key to `handle_receive_force_subtask_results()`.